aspnetcore/includes/razor.md

```diff
@@ -1,9 +1,3 @@
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a feature of the ASP.NET Core MVC framework that makes building and testing web UI easier and more productive. You can use Razor pages alongside controllers and views in the same project.
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point.
 
-We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC/Controller/Views version. The Razor Pages tutorial:
-
-* Is the preferred approach for new application development.
-* Is easier to follow.
-* Covers more features.
-
-If you choose this tutorial over the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version, let us know why in [this GitHub issue](https://github.com/aspnet/Docs/issues/6146).
+Developers who will be maintaining ASP.NET Core MVC projects should start with this tutorial.
```

Comment:
Is this statement really needed? Seems like it could be removed. I think there's value in simplifying here.

Reply:
That was an artifact to counter the bullet list that RP was better. 
removed. Ready to merge?